x-pack/platform/packages/shared/kbn-elastic-assistant/impl/connectorland/connector_selector_inline/action_type_selector_modal.tsx

```diff
@@ -29,7 +29,7 @@ interface Props {
   actionTypeSelectorInline: boolean;
 }
 const itemClassName = css`
-  inline-size: 220px;
+  inline-size: 150px;
```

Comment:
While this PR does not include the inference connector in the action type selector modal, it will in the future. Leaving this design change in to accommodate 4 connector types in the future